test/suite/test_metadata_cursor04.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-2020 MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest
+
+# test_metadata04.py
+#    Check metadata create cursors with complex tables.
+class test_metadata04(wttest.WiredTigerTestCase):
+    # Turn logging on for the database but we will turn logging off for all tables.
+    conn_config = 'log=(enabled)'
+    uri = 'table:metadata04'
+
+    def check_meta(self, uri, check_log):
+        cur = self.session.open_cursor('metadata:create', None, None)
+        cur.set_key(uri)
+        cur.search()
+        meta = cur.get_value()
+        cur.close()
+        self.pr("======== " +  uri)
+        self.pr(meta)
+        if not check_log:
+            return
+
+        if meta.find('log=(enabled=false') == -1:
+            self.pr(uri + " FAIL: " + meta)
+        else:
+            self.pr(uri + " SUCCESS")
+        self.assertTrue(meta.find('log=(enabled=false)') != -1)
```

Comment:
Instead of calling `meta.find('log=(enabled=false')` twice, might be better to introduce a temp variable, like a boolean log_found.

Reply:
Fixed.